packages/@aws-cdk-lib/gen.js

```diff
@@ -0,0 +1,230 @@
+/**
```

Comment:
To be clear: the code in this file is dogshit.

Reviewers, feel free to rip it apart.